src/App/Pages/Accounts/LoginPageViewModel.cs

```diff
@@ -140,6 +149,9 @@ public async Task InitAsync()
                 {
                     Email = await _stateService.GetRememberedEmailAsync();
                 }
+                var accountEmails = _stateService.AccountViews != null ? _stateService.AccountViews.Select(a => a.Email).ToList() : new List<string>();
+                SavedEmail = accountEmails.Contains(Email);
+
```

Comment:
Can be done as:
`SavedEmail = _stateService.AccountViews != null && _stateService.AccountViews.Any(e => e == Email);`

Reply:
fixed!